src/irr/asset/CGLILoader.cpp

```diff
@@ -0,0 +1,552 @@
+				case gl::INTERNAL_SRGB_BP_UNORM: return getTranslatedFinalFormat();				//GL_COMPRESSED_SRGB_ALPHA_BPTC_UNORM
+				case gl::INTERNAL_SRGB_PVRTC_2BPPV1: return getTranslatedFinalFormat();			//GL_COMPRESSED_SRGB_PVRTC_2BPPV1_EXT
+				case gl::INTERNAL_SRGB_PVRTC_4BPPV1: return getTranslatedFinalFormat();			//GL_COMPRESSED_SRGB_PVRTC_4BPPV1_EXT
+				case gl::INTERNAL_SRGB_ALPHA_PVRTC_2BPPV1: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV1_EXT
+				case gl::INTERNAL_SRGB_ALPHA_PVRTC_4BPPV1: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV1_EXT
+				case gl::INTERNAL_SRGB_ALPHA_PVRTC_2BPPV2: return getTranslatedFinalFormat();		//COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV2_IMG
+				case gl::INTERNAL_SRGB_ALPHA_PVRTC_4BPPV2: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV2_IMG
+				case gl::INTERNAL_SRGB8_ETC2: return getTranslatedFinalFormat();						//GL_COMPRESSED_SRGB8_ETC2
+				case gl::INTERNAL_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2: return getTranslatedFinalFormat();	//GL_COMPRESSED_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2
+				case gl::INTERNAL_SRGB8_ALPHA8_ETC2_EAC: return getTranslatedFinalFormat();			//GL_COMPRESSED_SRGB8_ALPHA8_ETC2_EAC
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_4x4: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_4x4_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_5x4: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_5x4_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_5x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_5x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_6x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_6x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_6x6: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_6x6_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_8x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_8x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_8x6: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_8x6_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_8x8: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_8x8_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x6: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x6_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x8: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x8_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x10: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x10_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_12x10: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_12x10_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_12x12: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_12x12_KHR
```

Comment:
These are all the same as https://github.com/buildaworldnet/IrrlichtBAW/pull/411/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.h&file-filters%5B%5D=No+extension#diff-28a7c4ae6aba8ec71c707eede79b01e9R477

Except that you swap `UNORM` for `SRGB`